subsystems/csr/csrlib/csrdll.spec

```diff
@@ -0,0 +1,17 @@
+@ stdcall CsrAllocateCaptureBuffer(long long)
```

Comment:
What is this file used for?
I don't see it referenced anywhere

Reply:
I added it for reference and later usage.